client/utils/__tests__/api-spec.js

```diff
@@ -42,19 +42,17 @@ describe('tranformCallDescriptor', () => {
 
   describe('request method', () => {
     it('uses GET by default', () => {
-      expect(apiAction({}).method).to.eq('GET')
+      expect(apiAction({}).method).toEqual('GET')
     })
 
     it('can override the request method', () => {
-      expect(apiAction({ method: 'POST' }).method).to.eq('POST')
+      expect(apiAction({ method: 'POST' }).method).toEqual('POST')
     })
   })
 
   describe('headers', () => {
     it('adds a content-type header', () => {
-      expect(apiAction({}).headers).to.contain({
-        'Content-Type': 'application/json'
-      })
+      expect(apiAction({}).headers['Content-Type']).toBe('application/json')
```

Comment:
Was reading up on Jest assertions last night.  Should this be `toEqual` to match the other string compares above, or should the others be `toBe`?